pkgs/tools/networking/openssh/default.nix

```diff
@@ -5,6 +5,7 @@
 , withGssapiPatches ? false
 , kerberos
 , linkOpenssl? true
+, sshKeysignPath ? "/run/wrappers/bin/ssh-keysign"
```

Comment:
I think this one should default to `null` and be passed in by the NixOS ssh module as an override.
That means non-nixos users get the current behaviour and NixOS users get the correct behaviour with the wrapper.

Reply:
@grahamc pointed out on irc that this is a bad idea:
>  this isn't workable I think, because _every_ ssh needs it set correctly, causing mass rebuilds for everybody

Reply:
I agree that we don't want a different openssh on NixOS.